bundles/org.openhab.binding.hue/README.md

```diff
@@ -13,12 +13,27 @@ There are two types of Hue Bridges, generally referred to as v1 (the rounded ver
 Only noticeable difference between the two generation of bridges is the added support for Apple HomeKit in v2.
 Both bridges are fully supported by this binding.
 
+Both types of bridges are accessed by means of the "CLIP" ('Connected Lighting Interface Protocol') Application Program Interface ('API').
+There are two versions of CLIP - namely CLIP v2 and CLIP v1.
+Philips has stated that any new features (such as dynamic scenes) will only be available on CLIP v2, and in the long term the CLIP v1 API will eventually be removed.
+Nevertheless it is possible that older v1 (round) bridges might not support CLIP v2, so you need to check that the bridge version is at least 1948086000.
+The CLIP v2 API is more modern and has more features, so it is recommended to use CLIP v2 for new openHAB installations.
```

Comment:
I don't think users can about an API being modern, but features they care about. :-) Perhaps we could mention push messages here? I would also revisit line 13 and rewrite/merge this, since that sentence is no longer correct.

Reply:
Ok.